src/OpenSage.Game/Graphics/MeshVertex.cs

```diff
@@ -27,10 +32,15 @@ public struct TexCoords
         {
             new VertexLayoutDescription(
                 new VertexElementDescription("POSITION", VertexElementSemantic.TextureCoordinate, VertexElementFormat.Float3),
+                new VertexElementDescription("POSITION", VertexElementSemantic.TextureCoordinate, VertexElementFormat.Float3),
+                new VertexElementDescription("NORMAL", VertexElementSemantic.TextureCoordinate, VertexElementFormat.Float3),
                 new VertexElementDescription("NORMAL", VertexElementSemantic.TextureCoordinate, VertexElementFormat.Float3),
```

Comment:
Not sure why this is all VertexElementSemantic.TextureCoordinate, shouldnt these be "Position" or sth?

Reply:
It’s correct as it is. Veldrid.SPIRV always maps the semantics as TEXCOORDn.

Reply:
ok, but then the other enum values make no sense...

Reply:
Veldrid can be used without spirv-cross or Veldrid.Spirv. Those enum values can be used, just not by us.